test/suite/test_txn24.py

```diff
@@ -44,6 +44,8 @@ class test_txn24(wttest.WiredTigerTestCase):
     ]
     scenarios = make_scenarios(table_params_values)
 
+    rollbacks_allowed = 0
```

Comment:
A comment would be helpful to indicate the reason for this setting (encouraging rollback error to occur and recorded in results.txt I believe?)

Reply:
Hi @lukech , I've added a comment explaining why rollbacks_allowed is now set to 0.